src/NzbDrone.Core/Applications/Radarr/RadarrV3Proxy.cs

```diff
@@ -116,6 +116,12 @@ public ValidationFailure TestConnection(RadarrIndexer indexer, RadarrSettings se
                     return new ValidationFailure("ProwlarrUrl", "Prowlarr url is invalid, Radarr cannot connect to Prowlarr");
                 }
 
+                if (ex.Response.StatusCode == HttpStatusCode.SeeOther)
```

Comment:
Don't we handle these now, at least I thought we did in some arrs?

<https://github.com/Radarr/Radarr/commit/88bc44f7ba7cd51c9e03065b472455e86eaa6d8e>

Reply:
Maybe it's just sonarr that we don't like following a redir?

Reply:
Idk, I'd have to test